docs/cli/ilab-local-config.md

```diff
@@ -0,0 +1,184 @@
+# InstructLab CLI Local Storage Proposal
+
+## Why is this needed?
+
+For tools to be useful to the user, they generally need to work on and know about data pertinent to the user. For example, podman and docker track image blobs, kubectl tracks cluster credentials, etc.
+
+The InstructLab CLI manages a number of settings today such as the teacher model, taxonomy location, inference model, etc. using a local file known as `config.yaml`.
+
+While this method allows ilab to avoid additional complexity of managing a dedicated directory, it’s limited by its ability to scale beyond constraining the user to run ilab in a single directory.
```

Comment:
```suggestion
While this method allows ilab to avoid additional complexity of managing a dedicated directory, it’s limited by its ability to scale beyond constraining the user to run `ilab` in a single directory.
```